lib/charts/simple_bar_chart.dart

```diff
@@ -45,7 +45,7 @@ mapChartData(Map<String, dynamic> labelValues, Map<String, dynamic> labelTitles)
   ];
 }
 
-/// Returns a color matching a [factor] between 0 and 1.
+/// Returns a color matching a [factor] between 0 and 1. 0 being green and 1 red.
```

Comment:
can't do suggestions in this mode;

'between 0 and 1, where 0 is green and 1 is red.

Reply:
```suggestion
/// Returns a color matching a [factor] between 0 and 1, where 0 is green and 1 is red.
```